packages/core-magistrate-transactions/src/handlers/bridgechain-update.ts

```diff
@@ -43,9 +43,11 @@ export class BridgechainUpdateTransactionHandler extends MagistrateTransactionHa
                     "business",
                 );
 
-                const { bridgechainId, seedNodes, ports } = transaction.asset.bridgechainUpdate;
-                businessAttributes.bridgechains[bridgechainId].bridgechainAsset.seedNodes = seedNodes;
-                businessAttributes.bridgechains[bridgechainId].bridgechainAsset.ports = ports;
+                const bridgechainUpdate = transaction.asset.bridgechainUpdate;
+                const bridgechainAsset =
+                    businessAttributes.bridgechains[bridgechainUpdate.bridgechainId].bridgechainAsset;
+                delete bridgechainUpdate.bridgechainId; // we don't want id in wallet bridgechain asset
```

Comment:
Instead of mutating the transaction asset, do it the other way around. Assign first and then delete the id from the attributes. It could introduce some unwanted side-effects.